src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportCoordinator.swift

```diff
@@ -167,7 +167,12 @@ final class ErrorReportsCoordinator: ErrorReportsCoordinating, RequiresAppDepend
 						self.topViewControllerViewModel?.updateViewModel(isHistorySectionIncluded: true)
 					case .failure(let error):
 						Log.error("ELS submission error: \(error)", log: .els, error: error)
-						self.showErrorAlert(with: error)
+						switch error {
+						case .otpError(let otpError):
+							otpError == .noNetworkConnection ? self.showErrorAlert(with: AppStrings.Common.noNetworkConnection) : self.showErrorAlert(with: error.localizedDescription)
```

Comment:
for the first point: the function is geting called twice one by passing an Error and the other by passing an OTPError So im passing the error strings directly, if i pass the OTPError we will need to type cast it again into OTPError since the function parameter is of Type Error

Reply:
regarding the error strings we are showing human readable message for no network errors, so it didn't make sense to add strings to all the enums if we only need one case or what do you think?